modules/related-posts/jetpack-related-posts.php

```diff
@@ -248,6 +253,127 @@ public function get_target_html_unsupported() {
 		return "\n\n<!-- Jetpack Related Posts is not supported in this context. -->\n\n";
 	}
 
+	/**
+	 * ===============
+	 * GUTENBERG BLOCK
+	 * ===============
+	 */
+
+	/**
+	 * Render the related posts markup.
+	 *
+	 * @param array $attributes Block attributes.
+	 * @return string
+	 */
+	public function render_block( $attributes ) {
+		$block_attributes = array(
+			'show_thumbnails' => isset( $attributes['displayThumbnails'] ) ? (bool) $attributes['displayThumbnails'] : false,
```

Comment:
```suggestion
			'show_thumbnails' => isset( $attributes['displayThumbnails'] ) && (bool) $attributes['displayThumbnails'],
```

Reply:
Nice, done in https://github.com/Automattic/jetpack/pull/10132/commits/c24f1010b7ba42fc683f81b3159be98310156060.